includes/class-wc-stripe-api.php

```diff
@@ -227,34 +227,33 @@ public static function request( $request, $api = 'charges', $method = 'POST', $w
 		);
 
 		$response_headers = wp_remote_retrieve_headers( $response );
-		// Log the stripe version in the response headers, if present.
-		if ( isset( $response_headers['stripe-version'] ) ) {
-			WC_Stripe_Logger::log( "{$api} response with stripe-version: " . $response_headers['stripe-version'] );
-		}
 
 		if ( is_wp_error( $response ) || empty( $response['body'] ) ) {
-			WC_Stripe_Logger::log(
-				'Error Response: ' . print_r( $response, true ) . PHP_EOL . PHP_EOL . 'Failed request: ' . print_r(
-					[
-						'api'             => $api,
-						'request'         => $request,
-						'idempotency_key' => $idempotency_key,
-					],
-					true
-				)
+			// Stripe redacts API keys in the response.
+			WC_Stripe_Logger::error(
+				"Stripe API error: {$method} {$api}",
+				[
+					'request'         => $request,
+					'idempotency_key' => $idempotency_key,
+					'response'        => ! empty( $response['body'] ) ? json_decode( $response['body'] ) : $response,
```

Comment:
We can also write this in a shorter and "more readable" way:
```suggestion
					'response'        => json_decode( $response['body'] ?? '' ) ?? $response,
```